### PR TITLE
Added most everything else from https://ethereum.consensys.net/ethere…

### PR DESCRIPTION
**Pull request by virgil**
…um-dev-portal

Still think we should add the images at some point.

### CODE REVIEW

File: docs/build/index.md [hunk 1]
```diff
@@ -87,13 +99,28 @@ Ethereum has a large and growing number of tools to help developers build, test,
...
+
+
 ### Web3-wrapper *Typescript alternative to Web3.js*
 - [Github](https://github.com/0xProject/0x-monorepo/tree/v2-prototype/packages/web3-wrapper)
 - [Documentation](https://0x.org/docs/web3-wrapper#introduction)
 
 ### Looking for other options?
 - [Ethereum Developer Tools List #Frontend-Ethereum-APIs](https://github.com/ConsenSys/ethereum-developer-tools-list#frontend-ethereum-apis)
 
+## Backend
```
Comment (ligi) [inline docs/build/index.md]: I really struggle with the term "Backend" here  - also having problems seeing why "Web3.py A Python implementation of Web3.js." is in Backend and Web3.js is frontend

Comment (virgil) [reply docs/build/index.md]: Organize it however you see fit.  I just copied the categories from the Consensys page.

**Comment by virgil** [comment]
My sole goal is to see the same content from the Consensys site on Ethereum.org.  I defer to the maintainer for how to organize the content.

**Comment by maurelian** [comment]
> My sole goal is to see the same content from the Consensys site on Ethereum.org

Is that a design goal for this site?

**Comment by virgil** [comment]
@maurelian The main goal is really just to obsolete https://ethereum.consensys.net/ethereum-dev-portal.  Everything that goes to the Consensys dev portal now goes to ethereum.org

**Comment by virgil** [comment]
Closing this.  Making several separate pull requests

**Pull request closed without merging**